content/blog/eng/v-protetto17-5-2024.md

```diff
@@ -0,0 +1,120 @@
+---
+
+authors: ["mgarza", "ana-radujko"]
+comments: true
+date: "2024-03-11 17:00:00"
+draft: false
+share: true
+categories: ['Protected Fridays']
+title: "Venerdì Protetto | May 2024 edition"
+description: "What happened in the last issue of Venerdì Protetto?"
+languageCode: "en-US"
+type: "post"
+twitterImage: '/images/social/social-preview-vp.png'
+twitterLargeImage: '/images/social/social-preview-vp.png'
+toc: false
+
+---
+
+
+Topics:
+
+
+During the last Venerdì Protetto (held on May 17, 2024), we had both a talk and a workshop. There they are:
+
+- [Hack the Box Workshop](#hack-the-box-workshop) by [Alessio Giorgianni](https://www.linkedin.com/in/alessio-giorgianni-b90500123/) (Facile.it)
+- [How to develop cognitive flexibility?](#how-to-develop-cognitive-flexibility) by [Arduino Mancini](https://www.linkedin.com/in/arduinomancini/) ([TIBICON](https://www.tibicon.net/))
+
+
+# Hack the box Workshop
+
+### What is this workshop? 
+After the successful experience of [having a code challenge in March 2024](https://engineering.facile.it/blog/eng/v-protetto8-3-2024/ "Previously, on Venerdì Protetto... a code challenge!"), we decided to go further in delivering experiences to our colleagues.
+
+Scouting around, I asked help to Alessio Giorgianni, a developer with a passion of white hat hacking competition. We agree upon trying an experience using a platform, called [Hack the Box](https://academy.hackthebox.com/). Hack the Box Academy offers lots of formation about IT Security and, in our case, some exercises we can use for hacking dummy application. We agree upon using a non-trivial quests, called [Jerrytok](https://www.hackthebox.com/achievement/challenge/48545/638). Jerrytok is a simple web-application, written in PHP, undisclosing the harmness of using template engines in not-proper way. It's a good introduction to SSTI, [Server Side Template Injection](https://portswigger.net/web-security/server-side-template-injection).
```

Comment:
- ...I asked Alessio Giorgiani, a developer with a passion for white hat hacking competitions, **for** help oppure ...I **turned to** Alessio Giorgiani, a developer with...
- ...with a passion **for**... 
- We agree**d** **to try**...
- using a platform called - toglierei la virgola
- formation? Si tratta di information oppure di training?
- for hacking dummy application**s** 
- We agree**d** upon using...
- **a** non-trivial quest oppure non trivial quest**s**
- ~~undisclosing~~ **which does not discolse** the ~~harmness~~ **harm** of using template engines in **a** not-proper way E' giusto che il senso sia che non rivela la pericolosità?

Reply:
HTB fornisce sia informazione che training, metto entrambi

Reply:
Il senso è l'opposto, cioè che rivela la pericolosità. Adatto le tue correzioni a riguardo